packages/block-library/src/navigation-link/index.php

```diff
@@ -5,77 +5,6 @@
  * @package WordPress
  */
 
-/**
- * Build an array with CSS classes and inline styles defining the colors
- * which will be applied to the navigation markup in the front-end.
- *
- * @param  array $context    Navigation block context.
- * @param  array $attributes Block attributes.
- * @return array Colors CSS classes and inline styles.
- */
-function block_core_navigation_link_build_css_colors( $context, $attributes ) {
```

Comment:
Since all functions in PHP are public, should we probably add `_deprecated_function( __FUNCTION__, '6.2' );` to this one and just call the other function from it, right?